src/molecules/side-menu.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from './side-menu.module.css';
+
+const SideMenu = ({ isOpen, closeMenu, subjects }) => {
+  return (
+    <>
+      {isOpen && <div className={`${styles.overlay} scrim`} onClick={closeMenu}></div>}
+      <div className={`${styles.sideMenu} surface`} data-open={isOpen}>
+        <div className={`${styles.topBar} on-surface`}>
+          <span className="title-small on-primary-text">出席確認システム</span>
+          <button className={`${styles.closeButton} on-primary`} onClick={closeMenu}>×</button>
```

Comment:
```suggestion
          <span className="title-small">出席確認システム</span>
          <button className={styles.closeButton} onClick={closeMenu}>×</button>
```
この primary 関係の色指定は不要そうです